latch_cli/services/stop_pod.py

```diff
@@ -0,0 +1,34 @@
+from urllib.parse import urljoin
+
+import click
+from latch_sdk_config.latch import NUCLEUS_URL
+
+from .. import tinyrequests
+from .cp.utils import get_auth_header
+
+
+def stop_pod(pod_id: int) -> None:
+    """Stops a pod given a pod_id"""
+    res = tinyrequests.post(
+        urljoin(NUCLEUS_URL, "/pods/stop"),
+        headers={"Authorization": get_auth_header()},
+        json={"pod_id": pod_id},
+    )
+
+    if res.status_code == 200:
+        click.secho(f"Pod with id `{pod_id}` Stopped.", fg="green")
+        return
+
+    if res.status_code == 403 or res.status_code == 404:
+        click.secho("Pod does not exist or permission denied.", fg="red")
+        return
```

Comment:
why not just separate the two

Reply:
these should always be grouped -- this endpoint is stupid and does not.